usecases/connect-autonomous-database/src/main/java/com/oracle/cloud/world/demo/repository/CountryRepository.java

```diff
@@ -0,0 +1,16 @@
+package com.oracle.cloud.world.demo.repository;
```

Comment:
Suggested to remove Cloud World reference (generalize)...

Reply:
Renamed